### DISCUSSION TITLE
RFC Call] Agenda - December 21, 2021

### DISCUSSION THREAD

**Discussion by jonathantneal** [withastro/roadmap#48] [Announcements]
# Agenda

- @sgruenholz2 https://github.com/withastro/rfcs/pull/58
- @natemoo-re https://github.com/withastro/rfcs/pull/51
- @FredKSchott https://github.com/withastro/rfcs/pull/49

--- 

_If there is something that you would like to see discussed, please comment below. We will prioritize pending RFC, but if we have time to spare we will open the call up to get attention on and discuss [other threads and feature requests.](https://github.com/withastro/rfcs/discussions)_
If there is something else you would like to discuss, please mention it here.

**Comment by matthewp** [comment]
3 for 3, success!
